mint-auditor/src/gnosis/sync.rs

```diff
@@ -0,0 +1,562 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Code for syncing transactions from the Gnosis API into the SQLite database.
+//!
+//! NOTE: Right now, if the audited safes configuration changes, one should
+//! delete the SQLite database and re-audit. The code is not smart enough to
+//! handle adding/removing safes/tokens for Gnosis transactions that were
+//! already processed.
+
+use super::{
+    api_data_types::{EthereumTransaction, MultiSigTransaction, RawGnosisTransaction, Transaction},
+    fetcher::GnosisSafeFetcher,
+    AuditedSafeConfig,
+};
+use crate::{
+    counters,
+    db::{Conn, GnosisSafeDeposit, GnosisSafeTx, GnosisSafeWithdrawal, MintAuditorDb},
+    error::Error,
+    gnosis::Error as GnosisError,
```

Comment:
Optional/Equivalent: Move this into the `super` group above.

Reply:
That would require `super::super::`, I'll move everything under `crate::`